Components/FlightControl/RocketSM.cpp

```diff
@@ -155,24 +153,23 @@ RocketState PreLaunch::HandleNonIgnitionCommands(RocketControlCommands rcAction,
         // Transition to abort state
         return RS_ABORT;
```

Comment:
This doesn't have to be included in this PR, but we should think about making the LAUNCH and DMB_ABORT input pins trigger an interrupt in order to immediately queue the right state transition command.